qiita_db/util.py

```diff
@@ -506,13 +506,15 @@ def compute_checksum(path):
     return crc & 0xffffffff
 
 
-def get_files_from_uploads_folders(study_id):
+def get_files_from_uploads_folders(study_id, ignore_hidden_and_folders=True):
```

Comment:
Tried to do that but the logic was a bit odd and decided to leave it as a single one. Note that we are not using anywhere this functionality and if we actually need it we can figure it out then.